src/tv2-common/videoSwitchers/TriCaster.ts

```diff
@@ -175,15 +189,96 @@ export class TriCaster extends VideoSwitcherBase {
 		)
 	}
 
-	public getDveTimelineObjects(_properties: DveProps): TSR.TSRTimelineObj[] {
-		throw new Error('Method not implemented.')
-	}
-
 	public updateUnpopulatedDveBoxes(
-		_timelineObject: TSR.TSRTimelineObj,
-		_input: number | SpecialInput
+		timelineObject: TSR.TSRTimelineObj,
+		input: number | SpecialInput
 	): TSR.TSRTimelineObj {
-		throw new Error('Method not implemented.')
+		if (!this.isDveBoxes(timelineObject)) {
+			this.logWrongTimelineObjectType(timelineObject, this.updateUnpopulatedDveBoxes.name)
+			return timelineObject
+		}
+
+		const layers: Partial<Record<TSR.TriCasterLayerName, TSR.TriCasterLayer>> = (
+			(timelineObject as TSR.TimelineObjTriCasterME).content.me as TSR.TriCasterMixEffectInEffectMode
+		).layers!
+
+		this.assignInputIfPlaceholder(layers.a!, input)
+		this.assignInputIfPlaceholder(layers.b!, input)
+		this.assignInputIfPlaceholder(layers.c!, input)
+		this.assignInputIfPlaceholder(layers.d!, input)
+		return timelineObject
+	}
+
+	public assignInputIfPlaceholder(layer: TSR.TriCasterLayer, input: number | SpecialInput): void {
+		const dveServerPlaceholder = 'input-1'
```

Comment:
What is the function of `dveServerPlaceholder` and why `'input-1'`? Just curious 😄